### PR TITLE
TMEDIA-238 - update all blocks to use translation keys

### PR DESCRIPTION
**Pull request by matthewroach**
## Description
Update all hard coded language strings to use intl.json files

* Updated the logo fallback alt to be a blank - which is a valid alt value. For assistive technologies a blank alt text denotes the image is for decorative purpose. Based on the blocks logic we have the means to set it correctly from the blocks.json file. If it's not set falling back to a value of "Placeholder logo" provides no value.

## Jira Ticket
- [TMEDIA-238](https://arcpublishing.atlassian.net/browse/TMEDIA-238)

## Acceptance Criteria
* Add new keys to Lokalise and set English value
* Update blocks code to use fusion:intl library to render text strings
* Update locale files to have all supported locale have English defaults

## Test Steps

1. Checkout this branch `git checkout TMEDIA-238-update-all-blocks-to-use-translation-keys`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/ads-block,@wpmedia/alert-bar-block,@wpmedia/article-body-block,@wpmedia/card-list-block,@wpmedia/footer-block,@wpmedia/full-author-bio-block,@wpmedia/global-phrases-block,@wpmedia/header-nav-chain-block,@wpmedia/links-bar-block,@wpmedia/numbered-list-block,@wpmedia/placeholder-image-block,@wpmedia/results-list-block,@wpmedia/search-results-list-block,@wpmedia/share-bar-block,@wpmedia/simple-list-block,@wpmedia/full-author-bio-block,@wpmedia/author-bio-block`
3. Validate English strings are still appearing in none English locale sites - you should not see the language string fallback
4. Sites 
    * EN - http://localhost/pf/all-blocks/?_website=the-gazette
    * FR - http://localhost/pf/all-blocks/?_website=the-sun
    * JA - http://localhost/pf/all-blocks/?_website=arc-demo-3

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [X] Confirmed all the test steps a reviewer will follow above are working.
- [X] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [X] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [X] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [X] Confirmed this PR has unit test files
  - [X] Ran `npm run test`, made sure all tests are passing
  - [X] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored).
- [X] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/article-body-block/chains/article-body/_children/table.jsx [hunk 1]
```diff
@@ -5,10 +5,10 @@ const Table = ({ element }) => {
   const { header, rows } = element;
 
   const tableHeaders = header.map((headerItem) => (
+    // eslint-disable-next-line jsx-a11y/control-has-associated-label
```
Comment (matthewroach) [inline blocks/article-body-block/chains/article-body/_children/table.jsx:8]: This is to account for the removal of the `aria-label` below as setting an `aria-label` here actually hinders accessibility as by default assistive technologies would use the contents of the table header whereas with an `aria-label` all table headers would have the value specified in the `aria-label`

File: blocks/author-bio-block/features/author-bio/default.jsx [hunk 2]
```diff
@@ -71,8 +62,14 @@ const renderAuthorInfo = (author, arcSite) => {
...
 const AuthorBioItems = () => {
   const { globalContent: content, arcSite } = useFusionContext();
+  const {
+    locale = 'en',
+  } = getProperties(arcSite);
+
+  const phrases = getTranslatedPhrases(locale);
   const { credits = {} } = content;
   const { by = [] } = credits;
+  const PrimaryColor = getThemeStyle(arcSite)['primary-color'];
```
Comment (JackHowa) [inline blocks/author-bio-block/features/author-bio/default.jsx:72]: misread this for a second, TitleCase? I'd expect camelCase

File: blocks/author-bio-block/features/author-bio/default.test.jsx [hunk 3]
```diff
@@ -350,14 +359,8 @@ describe('Given the list of author(s) from the article', () => {
     expect(socialButtonsContainer.children()).toHaveLength(13);
     const socialLinks = socialButtonsContainer.find('a');
     expect(socialLinks).toHaveLength(13);
-    socialLinks.forEach((link, index) => {
```
Comment (JackHowa) [inline blocks/author-bio-block/features/author-bio/default.test.jsx:353]: thanks for removing index

File: blocks/results-list-block/features/results-list/default.jsx [hunk 4]
```diff
@@ -381,12 +381,9 @@ class ResultsList extends Component {
           className="btn btn-sm"
           primaryFont={getThemeStyle(arcSite)['primary-font-family']}
           primaryColor={getThemeStyle(arcSite)['primary-color']}
+          aria-label={this.phrases.t('results-list-block.see-more-button-aria-label')}
         >
           {this.phrases.t('results-list-block.see-more-button')}
-          {' '}
```
Comment (JackHowa) [inline blocks/results-list-block/features/results-list/default.jsx:386]: this was removed?

**Comment by JackHowa** [comment]
Not sure if this is a regression with the dot separator for large promos or just an old branch 

Screen Shot 2021-06-14 at 11 13 52

**Review by JackHowa — APPROVED** [review]
minor codestyle suggestion. otherwise works as described

**Comment by matthewroach** [comment]
The dot separator is a bug which I filed here - https://arcpublishing.atlassian.net/browse/TMEDIA-269

File: blocks/results-list-block/features/results-list/default.jsx (hunk 4 above)
Comment (matthewroach) [reply blocks/results-list-block/features/results-list/default.jsx:386]: It was moved to be in an `aria-label` to allow language strings to not need special formatting

**Pull request merged**